roles/mig_controller_prereqs/tasks/prepare_remote_cluster.yml

```diff
@@ -1,10 +1,29 @@
 ---
+# OCP3 host clusters login to OCP4 for remote cluster tasks, the opposite for OCP4 host clusters.
+
 - name: Logging to OCP4 and saving credentials
   include_role:
     name: login_ocp
   vars:
     source_kubeconfig: "{{ playbook_dir }}/auth/kubeconfig"
     target_kubeconfig: "{{ ocp4_kubeconfig }}"
+  when: cluster_version == '3'
+
+- name: Logging to OCP3 and saving credentials
+  include_role:
+    name: login_ocp
+  vars:
+    source_kubeconfig: "{{ playbook_dir }}/kubeconfigs/kubeconfig"
+    target_kubeconfig: "{{ ocp3_kubeconfig }}"
```

Comment:
Seeing this behaviour makes me think that we may substitute the `ocp3_kubeconfig` and `ocp4_kubeconfig` to the `source_kubeconfig` and `target_kubeconfig`, this will be better in terms of current symmetrical behaviour.

Reply:
Yes sir, exactly what I was thinking when I circunvented the defaults with an explicit path